index.html

```diff
@@ -11,76 +11,153 @@
       <section id="my-recent-works">
-        <h2>My Recent Works</h2>
+        <h2 class="h2title">My Recent Works</h2>
         <hr class="hr">
       </section>
-      <section id="multi-post-stories" class="work">
-        <img id="gymfit" src="img/Placeholder.png" alt="Gymfit">
-        <h2 class="myh2bis">Multi-Post Stories</h2>
-        <p class="Inter-text">A daily selection of privately personalized reads;
-        no accounts or sign-ups required.
-        has been the industry's standard dummy text ever since the 1500s,
-        when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="boxed-words">
-          <li class="boxed-word">css</li>
-          <li class="boxed-word">html</li>
-          <li class="boxed-word">bootstrap</li>
-          <li class="boxed-word">Ruby</li>
-        </ul>
-        <a href="#multi-post-stories" class="button">See Project</a>
-      </section>
-      <section id="professional-art" class="work">
-        <h2 class="myh2tri">Professional Art Printing Data</h2>
-        <p class="Inter-text textart">A daily selection of privately personalized reads;
-        no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul class="boxed-words2">
-          <li class="boxed-word2">html</li>
-          <li class="boxed-word2">bootstrap</li>
-          <li class="boxed-word2">Ruby</li>
-        </ul>
-        <div class="buttoncontainer">
-          <a href="#professional-art" class="button">See Project</a>
-        </div>
-      </section>
-      <section id="professional-art2" class="work">
-        <h2 class="myh2tri">Professional Art Printing Data</h2>
-        <p class="Inter-text textart">A daily selection of privately personalized reads;
-        no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul class="boxed-words2">
-          <li class="boxed-word2">html</li>
-          <li class="boxed-word2">bootstrap</li>
-          <li class="boxed-word2">Ruby</li>
-        </ul>
-        <div class="buttoncontainer">
-          <a href="#professional-art2" class="button">See Project</a>
+      <section id="multi-post-stories" class="work1">
+        <img id="gymfit" src="img/ImgPlaceholder.png" alt="Gymfit">
+        <div id="textwrapper">
+          <h2 class="myh2bis">Multi-Post Stories</h2>
+          <p class="Inter-text firstInter-text">A daily selection of privately personalized reads;
+          no accounts or sign-ups required.
+          has been the industry's standard dummy text ever since the 1500s,
+          when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="boxed-words">
+            <li class="boxed-word">css</li>
+            <li class="boxed-word">html</li>
+            <li class="boxed-word">bootstrap</li>
+            <li class="boxed-word">Ruby</li>
+          </ul>
+          <a href="#multi-post-stories" class="button">See Project</a>
```

Comment:
- [ ] In this section after a certain width the styling of the elements inside this section starts to misplace please fix that.
Hint: you can do that by giving the width of this section a percentage value so it will be more responsive.

![image](https://user-images.githubusercontent.com/68796232/160031940-5fb7f751-f2cf-448e-9f9c-989814aada14.png)

Reply:
It looks better now. Still has the limitations of the "float" approach. Below, in the form, I tried the flex solution, arguably with better results. The figma design crops everything, we haven't any clue from what to expect. The right margin is there in the figma design. Hope my solution is good.